code/modules/antagonists/eldritch_cult/knowledge/void_lore.dm

```diff
@@ -0,0 +1,175 @@
+	banned_knowledge = list(/datum/eldritch_knowledge/rust_mark,/datum/eldritch_knowledge/ash_mark,/datum/eldritch_knowledge/flesh_mark)
+	route = PATH_VOID
+
+/datum/eldritch_knowledge/void_mark/on_mansus_grasp(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(isliving(target))
+		. = TRUE
+		var/mob/living/living_target = target
+		living_target.apply_status_effect(/datum/status_effect/eldritch/void)
+
+/datum/eldritch_knowledge/spell/void_phase
+	name = "Void Phase"
+	gain_text = "Reality bends under the power of memory, for all is fleeting, and what else stays?"
+	desc = "You gain a long range pointed blink that allows you to instantly teleport to your location, it causes aoe damage around you and your chosen location."
+	cost = 1
+	spell_to_add = /obj/effect/proc_holder/spell/pointed/void_blink
+	next_knowledge = list(/datum/eldritch_knowledge/rune_carver,/datum/eldritch_knowledge/crucible,/datum/eldritch_knowledge/void_blade_upgrade)
+	route = PATH_VOID
+
+/datum/eldritch_knowledge/rune_carver
+	name = "Carving Knife"
+	gain_text = "Etched, carved... eternal. I can carve the monolith and evoke their powers!"
+	desc = "You can create a carving knife, which allows you to create up to 3 carvings on the floor that have various effects on nonbelievers who walk over them, to create the carving knife transmute a knife with a glass shard and a piece of paper."
+	cost = 1
+	next_knowledge = list(/datum/eldritch_knowledge/spell/void_phase,/datum/eldritch_knowledge/summon/raw_prophet)
+	required_atoms = list(/obj/item/kitchen/knife,/obj/item/shard,/obj/item/paper)
+	result_atoms = list(/obj/item/melee/rune_knife)
+
+/datum/eldritch_knowledge/crucible
+	name = "Mawed Crucible"
+	gain_text = "This is pure agony, i wasn't able to summon the dereliction of the emperor, but i stumbled upon a diffrent recipe..."
+	desc = "Allows you to create a mawed crucible, eldritch structure that allows you to create potions of various effects, to do so transmute a table with a watertank"
+	cost = 1
+	next_knowledge = list(/datum/eldritch_knowledge/spell/void_phase,/datum/eldritch_knowledge/spell/area_conversion)
+	required_atoms = list(/obj/structure/reagent_dispensers/watertank,/obj/structure/table)
+	result_atoms = list(/obj/structure/eldritch_crucible)
+
+/datum/eldritch_knowledge/void_blade_upgrade
+	name = "Seeking blade"
+	gain_text = "Fleeting memories, fleeting feet. Not a single human can escape me now."
+	desc = "You can now use your blade on a distant marked target to move to them and attack them."
+	cost = 2
+	next_knowledge = list(/datum/eldritch_knowledge/spell/voidpull)
+	banned_knowledge = list(/datum/eldritch_knowledge/ash_blade_upgrade,/datum/eldritch_knowledge/flesh_blade_upgrade,/datum/eldritch_knowledge/rust_blade_upgrade)
+	route = PATH_VOID
+
+/datum/eldritch_knowledge/void_blade_upgrade/on_distant_eldritch_blade(atom/target, mob/user, proximity_flag, click_parameters)
```

Comment:
it'd be cool if this replaced the in-hand use effect of your void blade while you have a mark (maybe one from your blade instead of your mansus grasp?) active on them (this wouldn't consume your void blade). this would clear the mark when used, of course. you might have to remove the auto-hit on teleport for this alternate version to be balanced (or add a cooldown to the teleports from this).

so, like, you'd hit someone with your blade, then use it in-hand to teleport to them, then hit them again, teleport to them again, etc. and if you actually do need to escape, you can just press "z" to teleport to them, then immediately press z again without hitting them to use the normal blade in-hand effect.

Reply:
take it to coding feedback holy shit